src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -848,6 +848,13 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 		}
 	}
 
+	positiveIntegerOrNull(control: AbstractControl) {
```

Comment:
If it's blank the validation fails due to the ```positiveIntegerOrNull```function. I will clarify if the value should just default back to 0 if the field is blank

Reply:
Ok but then it again isn't *really* optional, right? Like, either we allow it to be blank and call it optional, or we don't allow it to be blank and inform the user that it can be set to 0 to disable it.